Added plotly as graphical output to info.rank, ceplane.plot, eib.plot, evi.plot, ceac.plot

AndreaBerardi opened the pull request:
Summary of edits:
- Added plot rendering via plotly (using graph="plotly") to:
  - ceplane.plot
  - eib.plot
  - ceac.plot
  - evi.plot
  - info.rank
- Included additional graphical output options to these functions, e.g. point_colors to ceplane.plot. These are fully documented in the function help pages.
- Upgraded version to 2.3-00
- Added rlang, dplyr to Imports to solve no visible binding note and for data manipulation functions in info.rank, respectively
- Added plotly to Suggests
- Aligned info.rank documentation to roxygen
- Updated documentation of modified functions

Notes:
- col and ICER.col objects now softly deprecated in ceplane.plot (point_colors and ICER_colors are preferred)

giabaio commented:
Thanks, Andrea --- all merged!
G


Gianluca Baio

Professor of Statistics & Health Economics
Department of Statistical Science
University College London
1-19 Torrington Place, London WC1E 6BT, UK
Telephone +44(0)20 7679 1248 (internal: 41248)
Fax +44 (0)20 7383 4703
Website: http://www.homepages.ucl.ac.uk/~ucakgba<http://www.homepages.ucl.ac.uk/~ucakgba/>
Blog: http:www.statistica.it/gianluca/blog


[https://images2.imgbox.com/17/20/W5mYrhtb_o.png]<https://twitter.com/gianlubaio>[https://images2.imgbox.com/94/69/jm87g9qO_o.png]<https://www.linkedin.com/in/gianluca-baio-b893879/>[https://images2.imgbox.com/30/46/FKyz04NR_o.png]<https://www.facebook.com/gianluca.baio.1>[https://images2.imgbox.com/25/61/30Cc1FIq_o.png]<https://github.com/giabaio>[https://images2.imgbox.com/82/0d/oKjooovM_o.png]<https://scholar.google.co.uk/citations?user=ro0QvGsAAAAJ&hl=en>[https://images2.imgbox.com/ad/72/iJ8zAJP0_o.png]<https://www.researchgate.net/profile/Gianluca_Baio>[https://images2.imgbox.com/16/40/RuU5VggG_o.png]<https://orcid.org/0000-0003-4314-2570>


On Wed, 27 Mar 2019 at 22:18, Andrea Berardi <notifications@github.com<mailto:notifications@github.com>> wrote:

Summary of edits:

  *   Added plot rendering via plotly (using graph="plotly") to:
     *   ceplane.plot
     *   eib.plot
     *   ceac.plot
     *   evi.plot
     *   info.rank
  *   Included additional graphical output options to these functions, e.g. point_colors to ceplane.plot. These are fully documented in the function help pages.
  *   Upgraded version to 2.3-00
  *   Added rlang, dplyr to Imports to solve no visible binding note and for data manipulation functions in info.rank, respectively
  *   Added plotly to Suggests
  *   Aligned info.rank documentation to roxygen
  *   Updated documentation of modified functions

Notes:

  *   col and ICER.col objects now softly deprecated in ceplane.plot (point_colors and ICER_colors are preferred)

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/giabaio/BCEA/pull/5

Commit Summary

  *   Included plotly version in info.rank function and updated package structure
  *   Added plotly option to eib.plot
  *   Update .gitignore
  *   Upgrade to v2.3-00
  *   Fixes to opacity, plot title and legend position
  *   Added plotly option to evpi.plot
  *   eib.plot and evi.plot documentation update
  *   Added plotly option to ceplane.plot and ceac.plot
  *   Fixed assignment error in ceplane.plot

File Changes

  *   A .gitignore<https://github.com/giabaio/BCEA/pull/5/files#diff-0> (5)
  *   A BCEA.Rproj<https://github.com/giabaio/BCEA/pull/5/files#diff-1> (18)
  *   M DESCRIPTION<https://github.com/giabaio/BCEA/pull/5/files#diff-2> (7)
  *   M NAMESPACE<https://github.com/giabaio/BCEA/pull/5/files#diff-3> (1)
  *   M R/ceac.plot.R<https://github.com/giabaio/BCEA/pull/5/files#diff-4> (449)
  *   M R/ceplane.plot.R<https://github.com/giabaio/BCEA/pull/5/files#diff-5> (961)
  *   M R/eib.plot.R<https://github.com/giabaio/BCEA/pull/5/files#diff-6> (444)
  *   M R/evi.plot.R<https://github.com/giabaio/BCEA/pull/5/files#diff-7> (222)
  *   M R/info.rank.R<https://github.com/giabaio/BCEA/pull/5/files#diff-8> (196)
  *   M man/ceac.plot.Rd<https://github.com/giabaio/BCEA/pull/5/files#diff-9> (110)
  *   M man/ceplane.plot.Rd<https://github.com/giabaio/BCEA/pull/5/files#diff-10> (181)
  *   M man/eib.plot.Rd<https://github.com/giabaio/BCEA/pull/5/files#diff-11> (156)
  *   M man/evi.plot.Rd<https://github.com/giabaio/BCEA/pull/5/files#diff-12> (76)
  *   M man/info.rank.Rd<https://github.com/giabaio/BCEA/pull/5/files#diff-13> (124)

Patch Links:

  *   https://github.com/giabaio/BCEA/pull/5.patch
  *   https://github.com/giabaio/BCEA/pull/5.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/giabaio/BCEA/pull/5>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AIUayUr1CPYSHdultN5yLk8jrYgqhp0Vks5va-5GgaJpZM4cO5BB>.

AndreaBerardi commented:
Happy to contribute – this was long overdue!

Andrea Berardi
Principal Consultant
PAREXEL International<http://www.parexel.com/>
T +44 (0) 207 121 1721

This communication, including any attachments, is intended only for the person or entity to which it is addressed and may contain confidential material. Any review, retransmission, distribution or other use of this information by persons or entities other than the intended recipient is prohibited. If you received this in error, please destroy any copies, contact the sender and delete the material from any computer.  Thank you.

From: Gianluca Baio <notifications@github.com>
Sent: 27 March 2019 22:38
To: giabaio/BCEA <BCEA@noreply.github.com>
Cc: Berardi, Andrea <Andrea.Berardi@PAREXEL.com>; Author <author@noreply.github.com>
Subject: Re: [giabaio/BCEA] Added plotly as graphical output to info.rank, ceplane.plot, eib.plot, evi.plot, ceac.plot (#5)

Thanks, Andrea --- all merged!
G


Gianluca Baio

Professor of Statistics & Health Economics
Department of Statistical Science
University College London
1-19 Torrington Place, London WC1E 6BT, UK
Telephone +44(0)20 7679 1248 (internal: 41248)
Fax +44 (0)20 7383 4703
Website: http://www.homepages.ucl.ac.uk/~ucakgba<http://www.homepages.ucl.ac.uk/~ucakgba/<http://www.homepages.ucl.ac.uk/~ucakgba%3chttp:/www.homepages.ucl.ac.uk/~ucakgba/>>
Blog: http:www.statistica.it/gianluca/blog


[https://images2.imgbox.com/17/20/W5mYrhtb_o.png]<https://twitter.com/gianlubaio>[https://images2.imgbox.com/94/69/jm87g9qO_o.png]<https://www.linkedin.com/in/gianluca-baio-b893879/>[https://images2.imgbox.com/30/46/FKyz04NR_o.png]<https://www.facebook.com/gianluca.baio.1>[https://images2.imgbox.com/25/61/30Cc1FIq_o.png]<https://github.com/giabaio>[https://images2.imgbox.com/82/0d/oKjooovM_o.png]<https://scholar.google.co.uk/citations?user=ro0QvGsAAAAJ&hl=en>[https://images2.imgbox.com/ad/72/iJ8zAJP0_o.png]<https://www.researchgate.net/profile/Gianluca_Baio>[https://images2.imgbox.com/16/40/RuU5VggG_o.png]<https://orcid.org/0000-0003-4314-2570>


On Wed, 27 Mar 2019 at 22:18, Andrea Berardi <notifications@github.com<mailto:notifications@github.com<mailto:notifications@github.com%3cmailto:notifications@github.com>>> wrote:

Summary of edits:

* Added plot rendering via plotly (using graph="plotly") to:
* ceplane.plot
* eib.plot
* ceac.plot
* evi.plot
* info.rank
* Included additional graphical output options to these functions, e.g. point_colors to ceplane.plot. These are fully documented in the function help pages.
* Upgraded version to 2.3-00
* Added rlang, dplyr to Imports to solve no visible binding note and for data manipulation functions in info.rank, respectively
* Added plotly to Suggests
* Aligned info.rank documentation to roxygen
* Updated documentation of modified functions

Notes:

* col and ICER.col objects now softly deprecated in ceplane.plot (point_colors and ICER_colors are preferred)

________________________________
You can view, comment on, or merge this pull request online at:

https://github.com/giabaio/BCEA/pull/5

Commit Summary

* Included plotly version in info.rank function and updated package structure
* Added plotly option to eib.plot
* Update .gitignore
* Upgrade to v2.3-00
* Fixes to opacity, plot title and legend position
* Added plotly option to evpi.plot
* eib.plot and evi.plot documentation update
* Added plotly option to ceplane.plot and ceac.plot
* Fixed assignment error in ceplane.plot

File Changes

* A .gitignore<https://github.com/giabaio/BCEA/pull/5/files#diff-0> (5)
* A BCEA.Rproj<https://github.com/giabaio/BCEA/pull/5/files#diff-1> (18)
* M DESCRIPTION<https://github.com/giabaio/BCEA/pull/5/files#diff-2> (7)
* M NAMESPACE<https://github.com/giabaio/BCEA/pull/5/files#diff-3> (1)
* M R/ceac.plot.R<https://github.com/giabaio/BCEA/pull/5/files#diff-4> (449)
* M R/ceplane.plot.R<https://github.com/giabaio/BCEA/pull/5/files#diff-5> (961)
* M R/eib.plot.R<https://github.com/giabaio/BCEA/pull/5/files#diff-6> (444)
* M R/evi.plot.R<https://github.com/giabaio/BCEA/pull/5/files#diff-7> (222)
* M R/info.rank.R<https://github.com/giabaio/BCEA/pull/5/files#diff-8> (196)
* M man/ceac.plot.Rd<https://github.com/giabaio/BCEA/pull/5/files#diff-9> (110)
* M man/ceplane.plot.Rd<https://github.com/giabaio/BCEA/pull/5/files#diff-10> (181)
* M man/eib.plot.Rd<https://github.com/giabaio/BCEA/pull/5/files#diff-11> (156)
* M man/evi.plot.Rd<https://github.com/giabaio/BCEA/pull/5/files#diff-12> (76)
* M man/info.rank.Rd<https://github.com/giabaio/BCEA/pull/5/files#diff-13> (124)

Patch Links:

* https://github.com/giabaio/BCEA/pull/5.patch
* https://github.com/giabaio/BCEA/pull/5.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/giabaio/BCEA/pull/5>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AIUayUr1CPYSHdultN5yLk8jrYgqhp0Vks5va-5GgaJpZM4cO5BB>.


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/giabaio/BCEA/pull/5#issuecomment-477373975>, or mute the thread<https://github.com/notifications/unsubscribe-auth/Ap2LJNHdRrHCfP3Pu9v9WgWACGK8T3C5ks5va_LTgaJpZM4cO5BB>.
